src/DashboardView/Diagram/PieChart/PieChart.js

```diff
@@ -29,6 +29,7 @@ class PieChart extends Component {
     componentDidMount() {
         const charWrapper = this.refs.canvas;
         if(charWrapper) {
+            const fontSize = Math.max((12/1920) * window.innerWidth, 15);
```

Comment:
Not possible here.
We are not able to style canvas elements using css and breakpoins. Therefore I calculated the font-size in javascript.